src/components/pip/pip-child.tsx

```diff
@@ -53,10 +53,10 @@ export class PipChild extends Component<PIPChildComponentProps> {
     return (
       <div>
         <div className={styles.innerButtons}>
-          <Button className={styles.iconContainer} onClick={onInversePIP}>
+          <Button className={styles.iconContainer} onClick={onInversePIP} tooltip={{label: 'Switch Screen', type: 'bottom-left'}}>
```

Comment:
Make a translations folder, create en.i18n.json and put all texts there. Even though we don't have localisation for this repo lets at least have all texts in one place to it will be easily handled in the future.
@yairans, do we somehow support localisation of plugins by the core?

Reply:
yes. localization is supported for injected components out of the box

Reply:
> Make a translations folder, create en.i18n.json and put all texts there. Even though we don't have localisation for this repo lets at least have all texts in one place to it will be easily handled in the future.
> @yairans, do we somehow support localisation of plugins by the core?

We already discussed it with @yairans 
Translations gonna be handled as a separate story, @yairans will create ticket for it.